code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -4405,6 +4405,33 @@
 
 	M.nutrition += nutriment_factor
 
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		switch(volume)
+			if(1 to 15)
+				if(prob(5))
+					to_chat(H,"<span class='warning'>You burp.</span>")
```

Comment:
in space, no-one can hear you burp

Reply:
make this a `visible_message` or `emote`